pages/common/vue-build.md

```diff
@@ -0,0 +1,8 @@
+# vue build
+
+> A subcommand provided by `@vue/cli` and `@vue/cli-service-global` that allows you to do painless prototyping.
```

Comment:
Let us change "you to do painless" with "quick" to keep it formal and impartial.

Reply:
Thanks. Making the changes.